### PR TITLE
Setup GitHub actions

### PR DESCRIPTION
**Pull request by arkiaman**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on: pull_request
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Unshallow
```
Comment (twsouthwick) [inline .github/workflows/release.yml]: You should be able to add `fetch-depth: 0` to the `Checkout` step instead

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Visit https://goreleaser.com for documentation on how to customize this
```
Comment (twsouthwick) [inline .goreleaser.yml:1]: cool - this simplifies some of the go config!

File: .goreleaser.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      # usage by users in CI/CD systems like Terraform Cloud where
+      # they are unable to install libraries.
+      - CGO_ENABLED=0
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    flags:
+      - -trimpath
+    ldflags:
+      - "-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}"
+    goos:
+      - windows
```
Comment (twsouthwick) [inline .goreleaser.yml:20]: may be good to add darwin here so mac users can get a build

**Review by twsouthwick — APPROVED** [review]
Looks great! goreleaser is pretty slick. Do you still need the .sh files you used? You may be able to use goreleaser locally as well so you can just remove them

**Comment by arkiaman** [comment]
fixes #12

**Pull request merged**